lms/services/canvas_api/_basic.py

```diff
@@ -47,6 +47,7 @@ def send(
         method,
         path,
         schema,
+        timeout,
```

Comment:
I added `timeout` here as a non-optional argument with no default value. `AuthenticatedClient` (above) is the only thing that calls `BasicClient` so it makes sense for the default `timeout` to live in `AuthenticatedClient` (otherwise I'd have to duplicate it). So I changed `AuthenticatedClient` to always pass `timeout` when calling `BasicClient.send()`